src/NodeJS/Server.php

```diff
@@ -194,6 +202,46 @@ public function getNodeModulesPath()
     }
 
     /**
+     * Get a single option.
+     *
+     * @param string $option The option name to retrieve.
+     * @param mixed $default_value The value to return if the option is not set.
+     *
+     * @return mixed The option value or default value if it is not set.
+     */
+    public function getOption($option, $default_value = NULL)
```

Comment:
Please remove this method.

I don't see any potential use of this method, because the only consumer of this options is Zombie itself. Nobody wants to introspect what options current server has.